kolibri/plugins/learn/assets/src/views/ContentCard/index.vue

```diff
@@ -146,6 +198,17 @@
         context:
           'Some Kolibri resources may be duplicated in different topics or channels.\n\nSearch results will indicate when a resource is duplicated, and learners can click on the "...locations" link to discover the details for each location.',
       },
+      viewInformation: {
+        message: 'View information',
+      },
+      moreOptions: {
+        message: 'More options',
+      },
+      logo: {
```

Comment:
Thank you for thinking about this, but check the comment above around how the channel thumbnail is presented: if it is placed adjacent to the visible string with channel title, and is just going to read the channel title (again), it's redundant information and we can spare the user that 😉.